static/js/formatters-internal.js

```diff
@@ -566,3 +566,25 @@ export function getYoutubeUrl(videos = []) {
     : null;
   return youtubeVideoUrl;
 }
+
+/**
+ * construct a string of displayable category names based on given category ids from liveAPI
+ * and a mapping of category ids to names.
+ * 
+ * @param {string[]} categoryIds category ids from liveAPI
+ * @param {Object[]} categoryMap mapping of category ids to names
+ * @param {string} categoryMap[].id id of a category entry
+ * @param {string} categoryMap[].category name of a category entry
+ * @returns {string} category names separated by comma
+ */
+ export function getCategoryNames(categoryIds, categoryMap) {
+  if (!categoryIds || !categoryMap) {
+    return '';
+  }
+  const categoryNames = categoryIds.reduce((list, id) => {
+    const categoryEntry = categoryMap.find(category => category.id === id);
+    categoryEntry ? list.push(categoryEntry.category) : console.error(`Unable to find category name for id ${id}.`);
+    return list;
+  }, []);
+  return categoryNames.join(', ');
```

Comment:
are they going to ask for the last element to be separated with an "and" instead of a comma?
like would they ever want
`"category 1, category2, and category3`
(I hope not because i18n would be annoying with this)

Reply:
good point. Talked to Rose, I will revert this to return an array of category instead, and it will be up to the user to customize it with join list formatter or manipulate the list however they want